packages/components/WhyItWorksSection.tsx

```diff
@@ -58,38 +58,35 @@ interface CardItemProps {
 function CardItem({ title, icon, description }: CardItemProps) {
   return (
     <Box>
-      <Icon
-        as={icon}
-        w="2rem"
-        h="2rem"
-        marginBottom="1rem"
-        color="purple.500"
-      />
-      <Heading size="sm" marginBottom="1rem">
+      <Icon as={icon} w="10rem" h="10rem" m="1rem" />
+      <Heading size="md" fontWeight={400} pb="1rem">
         {title}
       </Heading>
-      <Text color="gray.500">{description}</Text>
+      <Text color="gray.500" marginTop="1rem" maxWidth="30rem">
```

Comment:
Acho que podemos tirar o `maxWidth` e deixar a margin no `Heading`.

```suggestion
      <Text color="gray.500" >
```